### DISCUSSION TITLE
is normal to wait 6 WEEKS  for git hub support  to reinstate my account.

### DISCUSSION THREAD

**Discussion by hugo88144** [community/community#144013] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

AHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH.

**Comment by Hyphonical** [comment]
Highly likely, GitHub is currently busy addressing everyone's inquiries, so it might take a bit of time to solve your problem. While I've experienced some delays in contacting GitHub in the past, all my issues were eventually resolved.
